### PR TITLE
Proxy fixes

### PR DESCRIPTION
**Pull request by gardnk**
I updated two javascript commands to use ```number``` and not ```{}```

Also changed enum values from string to number so that it builds, but I'm not sure what numbers to use so need some help there @woksin please and thank you

### CODE REVIEW

File: Source/Reporting/Web/src/app/Reporting/CaseReportsForListing/filtering/filter.pipe.ts [hunk 1]
```diff
@@ -2,10 +2,10 @@ import {Pipe, PipeTransform} from '@angular/core';
 import {CaseReportForListing} from '../CaseReportForListing';
 
 enum CaseReportStatus {
-    Success = "Success",
-    TextMessageParsingError = "TextMessageParsingError",
-    UnknownDataCollector = "UnknownDataCollector",
-    TextMessageParsingErrorAndUnknownDataCollector = "TextMessageParsingErrorAndUnknownDataCollector"
+    Success = 0,
```
Comment (gardnk) [inline Source/Reporting/Web/src/app/Reporting/CaseReportsForListing/filtering/filter.pipe.ts]: I don't know what numbers to use here 🤷‍♂

Comment (woksin) [reply Source/Reporting/Web/src/app/Reporting/CaseReportsForListing/filtering/filter.pipe.ts]: The enum value definitions can be found in the backend c# code with the same names

Look in the Concepts project and the Readmodels project for enum definitions

Comment (gardnk) [reply Source/Reporting/Web/src/app/Reporting/CaseReportsForListing/filtering/filter.pipe.ts]: Ok thanks, looks like it was correct

**Comment by gardnk** [comment]
This runs like a charm on my end, can you review @woksin ?

**Pull request merged**